src/playbook.yaml

```diff
@@ -13,6 +13,9 @@
         name:
           - "ntpdate"
           - "curl"
+          - "python2.7-minimal"    # for legacy apps
```

Comment:
Doesn't the aws ubuntu base image come with `Python2.7`?

Reply:
nope, when i tried to run python2.7 it says i should install `python2.7-minimal` and the cnf tool only works with python2.7.   this seems to be the most accurate info: https://askubuntu.com/a/1104401